MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Muon/MuonAnalysis.h

```diff
@@ -249,6 +249,9 @@ private slots:
   /// Called when "enable multi fit" checkbox is turned on/off
   void multiFitCheckboxChanged(int state);
 
+  /// Called when "TF Asymmetry" checkbox is turned on/off
+  void TFAsymmCheckboxChanged(int state);
+  void setTFAsymm(Muon::TFAsymmState);
```

Comment:
Also if this function should be used to set the state of TF Asymmetry checkbox, could we make all calls that change the state go through it?

Independent calls (direct calls to the presenter function):
`MuonAnalysis.cpp`: 
  - Line 2118
  - Line 3047
  - Line 3071